logic/src/commonMain/kotlin/com/wire/kalium/logic/data/id/QualifiedId.kt

```diff
@@ -1,8 +1,16 @@
 package com.wire.kalium.logic.data.id
 
+import com.wire.kalium.network.tools.KtxSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+@Serializable
 data class QualifiedID(
     val value: String,
     val domain: String
-)
-
-typealias ConversationId = QualifiedID
+) {
+    fun toJson(): String = Json.encodeToString(this)
```

Comment:
Thanks @gongracr, I have a suggestion, what do you think of making these functions as extensions and generic, so we can access them for all of data model? since we need them in most of models that are playing as arguments between our screens.

Reply:
> So we can access them for for all of data model

If it is generic, how could we be sure that the class we are calling `.toJson()` is actually annotated with `@Serializable`?

From all the data that `logic` exposes, just a handful of classes have this need of being serialised from/to JSON. Mainly IDs I guess.